src/XPurchase/VirtoCommerce.XPurchase/CartAggregate.cs

```diff
@@ -515,6 +516,14 @@ public virtual async Task<CartAggregate> RecalculateAsync()
             return this;
         }
 
+        public virtual Task<CartAggregate> SetItemFulfillmentCenterAsync(LineItem lineItem, CartProduct cartProduct)
```

Comment:
No reason to make it Task.

Reply:
@Vectorfield4 This is for a potential implementation override.